typescript/db/schema.prisma

```diff
@@ -153,296 +162,7 @@ model VerificationToken {
 }
 
 /////////////////////////////////////////////////////////////////////////////
-//////////////////////            Strapi          ///////////////////////////
```

Comment:
Why did you remove all the Strapi related schema?

Reply:
Because I found out that it's possible ! It was always very annoying but I found a way to keep it in the DB but remove it from here so it's cool